torchtext/experimental/datasets/text_classification.py

```diff
@@ -87,32 +91,44 @@ def get_vocab(self):
         return self.vocab
 
 
-def _setup_datasets(dataset_name, root='.data', ngrams=1, vocab=None,
-                    tokenizer=None, data_select=('train', 'test')):
+def _setup_datasets(
+    dataset_name,
+    root=".data",
+    ngrams=1,
+    vocab=None,
+    tokenizer=None,
+    data_select=("train", "test"),
+):
     text_transform = []
     if tokenizer is None:
-        tokenizer = get_tokenizer('basic_english')
+        tokenizer = get_tokenizer("basic_english")
     text_transform = squential_transforms(tokenizer, ngrams_func(ngrams))
 
     if isinstance(data_select, str):
         data_select = [data_select]
-    if not set(data_select).issubset(set(('train', 'test'))):
-        raise TypeError('Given data selection {} is not supported!'.format(data_select))
+    if not set(data_select).issubset(set(("train", "test"))):
+        raise TypeError("Given data selection {} is not supported!".format(data_select))
     train, test = DATASETS[dataset_name](root=root)
     # Cache raw text iterable dataset
-    raw_data = {'train': [(label, txt) for (label, txt) in train],
-                'test': [(label, txt) for (label, txt) in test]}
+    raw_data = {
+        "train": [(label, txt) for (label, txt) in train],
+        "test": [(label, txt) for (label, txt) in test],
+    }
 
     if vocab is None:
-        if 'train' not in data_select:
+        if "train" not in data_select:
             raise TypeError("Must pass a vocab if train is not selected.")
-        vocab = build_vocab(raw_data['train'], text_transform)
-    text_transform = squential_transforms(text_transform, vocab_func(vocab),
-                                          totensor(dtype=torch.long))
+        vocab = build_vocab(raw_data["train"], text_transform)
+    text_transform = squential_transforms(
+        text_transform, vocab_func(vocab), totensor(dtype=torch.long)
+    )
     label_transform = squential_transforms(totensor(dtype=torch.long))
-    return tuple(TextClassificationDataset(raw_data[item], vocab,
-                                           (label_transform, text_transform))
-                 for item in data_select)
+    return tuple(
+        TextClassificationDataset(
+            raw_data[item], vocab, (label_transform, lambda x: None)
```

Comment:
99% of all changes in this file are due to the linter I ran so it's guaranteed to pass the lint tests. But this change here is manual. I'm setting the text_transforms to a function that returns None. The expectation is that this change will cause various tests to fail.